packages/cli/test/metadata/should-autoinstall.test.ts

```diff
@@ -0,0 +1,38 @@
+import test from 'ava';
+import {shouldAutoinstall} from '../../src/metadata/handler'
+
+test('should return false when no adaptor is provided', (t) => {
+  const adaptor = '';
+  const result = shouldAutoinstall(adaptor);
+  t.is(result, false);
```

Comment:
You should replace `t.is(result, false)` with `t.false(result)`. It's a bit more concise and we'll get a nicer error on fail.